package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "build": "grunt es6transpiler replace umd uglify",
     "jsdoc": "jsdoc -c .jsdoc",
-    "test": "grunt mochaTest"
+    "test": "npm run build && grunt mochaTest"
```

Comment:
To generate the `dist` folder, which I had to change the moment-range require path to use to avoid syntax errors.